pages/common/ffe.md

```diff
@@ -0,0 +1,29 @@
+# ffe
+
+> Extract fields from a flat database file and write to another format.
+> `ffe` needs a text config file to interpret the input and format the output.
+> More information: <http://ff-extractor.sourceforge.net/ffe.html>.
+
+- Display all input data using the specified data configuration:
+
+`ffe --configuration={{path/to/config.ffe}} {{path/to/input}}`
+
+- Convert an input file to an output file in a new format:
+
+`ffe --output={{path/to/output}} -c {{path/to/config.ffe}} {{path/to/input}}`
+
+- Select input structure and print format from definitions in `~/.fferc` config file:
+
+`ffe --structure={{structure}} --print={{format}} {{path/to/input}}`
```

Comment:
What is "format" in this instance?

Reply:
Both {{structure}} and {{format}} are defined in the (not shown) configuration file.